tests/plugins/subcommands/doctor/test_health_checks.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from pathlib import Path
+from conda.plugins.subcommands.doctor import health_checks
+import json
+import pytest
+
+BIN_TEST_EXE = "bin/test-exe"
+LIB_TEST_PACKAGE = "lib/test-package.py"
+TEST_PACKAGE_JSON = "test-package.json"
+
+PACKAGE_JSON = {"files": [BIN_TEST_EXE, LIB_TEST_PACKAGE]}
+
+PACKAGE_JSON_WITH_MISSING_FILES = {"files": [BIN_TEST_EXE, LIB_TEST_PACKAGE, "missing.py"]}
+
+
+@pytest.fixture()
+def conda_mock_dir(tmpdir):
+    """Fixture that returns a testing environment with no missing files"""
```

Comment:
I'm surprised we don't have a fixture yet for this, would you mind double-checking if there aren't tools to create these folders elsewhere, and if not, move them into `conda.testing.fixtures`?

Reply:
yes there are alternatives to this, I'm actively working to revamp these helper functions here:

- https://github.com/conda/conda/pull/12592

so I'm thinking we leave this as is until that effort is completed